src/Auth_Command.php

```diff
@@ -717,18 +886,38 @@ public function list( $args, $assoc_args ) {
 				if ( empty( $auths ) ) {
 					EE::warning( sprintf( 'Auth does not exists on %s', $site_url ) );
 				} else {
-					$formatter = new EE\Formatter( $assoc_args, [ 'username', 'password' ] );
+					$formatter = new EE\Formatter( $assoc_args, array( 'username', 'password' ) );
 					$formatter->display_items( $auths );
 				}
 			}
 			if ( ! empty( $log_msg ) ) {
 				EE::log( PHP_EOL . $log_msg );
 			}
 			if ( ! empty( $auths_global ) ) {
-				$formatter = new EE\Formatter( $assoc_args, [ 'username', 'password' ] );
+				$formatter = new EE\Formatter( $assoc_args, array( 'username', 'password' ) );
 				$formatter->display_items( $auths_global );
 			}
 		}
 	}
-}
 
+	/**
+	 * Helper function for ee auth list admin-tools
+	 * Prints all the auths on `admin-tools`
+	 *
+	 * @return void
+	 */
+	private function admin_tools_list_auth() {
+		$auths = $this->get_auths( 'default_admin_tools', false, false );
+
+		if ( empty( $auths ) ) {
+			EE::warning( sprintf( 'Auth does not exists on `default_admin_tools`' ) );
```

Comment:
Change it to error for consistency with rest of the codebase i.e. site list etc..

Reply:
Resolved all the occurences (https://github.com/EasyEngine/auth-command/pull/52#discussion_r712895606).